text_formatter.go

```diff
@@ -9,23 +9,44 @@ import (
 	"time"
 )
 
+// NoColor: reset; clears all colors and styles (to white on black)
+// Red: set foreground color to red
+// Green: set foreground color to green
+// Yellow: set foreground color to yellow
+// Blue: set foreground color to blue
+// Magenta: set foreground color to magenta
+// Cyan: set foreground color to cyan
+// Gray: set foreground color to gray
+// Grey: set foreground color to gray (UK)
```

Comment:
true but otherwise the linter will complain if public constants have no description